gap/almostsimple.gi

```diff
@@ -17,6 +17,7 @@
 ##
 #############################################################################
 
+# parse specific string into number
```

Comment:
What is a "specific" string? Perhaps you meant "the specified string <number>"? But looking at the code, it can also be an int?

(While I am at it: What is `d`, what what is `default`? But of course it is fair if you don't want to answer that :-)